Tests/NetworkServiceTests.swift

```diff
@@ -32,15 +32,16 @@ class NetworkServiceTests: XCTestCase {
     
     func test_SuccessResponseWithNoData_GeneratesNoDataError() {
```

Comment:
Test name will need to be updated.

Reply:
The name does not need to be changed anymore now that `noData` generates an error again